Tella/Data/Database/Common/SQLStatementBuilder.swift

```diff
@@ -30,6 +30,23 @@ class SQLiteStatementBuilder {
         return userVersion
         
     }
+    
+    func columnIsExist(tableName: String, column: String) -> Bool {
+        
+        do {
+            guard let selectStatement = try prepareStatement(sql: "PRAGMA table_info(\(tableName))") else {
+                throw RuntimeError(errorMessage)
```

Comment:
maybe use String.format for better readability? 
` let statement = String(format: "PRAGMA table_info(%@)", tableName)`
and to  to avoid string concatenation problems?